src/pybind/mgr/dashboard_v2/module_standby.py

```diff
@@ -0,0 +1,62 @@
+import cherrypy
```

Comment:
I don't think it's worth to create a new python file for this. Please move the code to module.py.

Reply:
@sebastian-philipp objections?

Reply:
There are two reasons for creating a new Python file:

1. `StandbyModule` is self-contained and doesn't need any code form `module.py`. There is no need to import any other Python modules e.g. controllers in a standby mgr.
2. Looks like the logger needs to be a different one. Having this code in `module.py` would result in an ugly second logger definition.

Reply:
in the dashboard module (aka dashboard_v1) the standbymodule is declared in `module.py` along side with mgrmodule.

I don't see why we need a change in the logger.

Reply:
Even though it was decided to move this class into module.py, I disagree with this decision. Having this class in a separate module makes sense. As long as we need a different `logger` definition. 

Using the module namespace to differentiate between loggers makes sense.